srcpkgs/python-PyAudio/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'python-PyAudio'
+pkgname=python-PyAudio
+version=0.2.11
+revision=1
+noarch=yes
+wrksrc="PyAudio-${version}"
+build_style=python-module
+pycompile_module="PyAudio.py"
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel portaudio-devel"
+depends="python"
+short_desc="Python2 Bindings for PortAudio, a cross-platform audio IO library"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="MIT"
+homepage="https://people.csail.mit.edu/hubert/pyaudio/"
+distfiles="${PYPI_SITE}/P/PyAudio/PyAudio-${version}.tar.gz"
+checksum=93bfde30e0b64e63a46f2fd77e85c41fd51182a4a3413d9edfaf9ffaa26efb74
+
+post_install() {
+	vlicense "${FILESDIR}/LICENSE"
+}
+
+python3-PyAudio_package() {
+	noarch=yes
+	depends="python3"
```

Comment:
Ditto to both `noarch` and `depends`.
Also, add `pycompile_module="pyaudio.py"` here as well.